server/utils/nunjucksSetup.ts

```diff
@@ -16,7 +17,7 @@ type Error = {
   text: string
 }
 
-export default function (app, path) {
+export default function configureNunjucks(app: Express.Application): void {
```

Comment:
fixing some type issues here and noticed path didn't need to be injected as a dependency